src/query/graphite/native/builtin_functions.go

```diff
@@ -94,6 +94,45 @@ func sortByMaxima(ctx *common.Context, series singlePathSpec) (ts.SeriesList, er
 	return highestMax(ctx, series, len(series.Values))
 }
 
+/*
```

Comment:
nit: mind using `//` comments instead of `/**/`?

Reply:
done!